docs/misc/prune-example.txt

```diff
@@ -1,21 +1,38 @@
 borg prune visualized
 =====================
 
-To understand prune, it helps to substitute the term `prune` with `delete all backups but keep the latest backup of the x and those of the last nx's that have backups`, and then simply replace `x` with the time frame (secondly, minutely, hourly, daily, weekly, monthly, or yearly), and replace `n` with a number (1, 2, 3, etc.). 
-So `prune --keep-daily 7` means `delete all backups but keep the latest backup of the day and those of the last 7 days that have backups`.
-Similarly, `--keep-monthly 6` means `delete all backups but keep the latest backup of the month and those of the last 6 months that have backups`.
-
-Assume you started using borg on 2015-01-01.
+To understand prune, it helps to substitute the term `prune --keep` with `delete all backups but keep the latest backup of each of the last 'n' INTERVALS (secondly, minutely, hourly, daily, weekly, monthly, or yearly), including the present, that have backups. 
+So `prune --keep-daily 7` means `delete all backups but keep the latest backup of each of the last 7 DAYS (including today) that have backups`.
+Note that it's not "keep the latest backup on each of the last 7 days," but rather "keep the latest backup for each of the last 7 days _that have backups._"
+For otherwise it would mean that if backups weren't taken on two days in the last 7 days, you'd be left with only 5 backups after running prune. 
+Such a scenario, however, will play out only if there aren't any backups before the last 7 days. 
+But if backups are available, then prune will be leave you with 7 backups.
```

Comment:
Hmm, this is repeating 3 times somehow - you think that is necessary to explain it?

We can keep it if you do, considering this is a separate file of the docs anyway, only read by people searching for more explanations...

Reply:
I'm not a coder. And I found borg docs difficult to understand. I had to spend a lot of time on IRC and Reddit trying to understand what the docs were really saying.

So, I've written it for someone like me (maybe not very sharp). Hopefully, the goal is to get everyone on the borg standard.

But I've rephrased it to be more concise.

Reply:
ok, thanks!